Makefile

```diff
@@ -655,6 +655,31 @@ bundle: generate manifests kustomize ## Generate bundle manifests and metadata,
 		yq -riSY  '(.spec.install.spec.deployments[0].spec.template.spec.containers[0].securityContext."runAsNonRoot") = true' "$${NEW_CSV}"
 	fi
 
+	# set InstallMode for next-all-namespaces
+	if [[ $${channel} == 'next-all-namespaces' ]];then
+		yq -Yi '.spec.installModes[] |= if .type=="OwnNamespace" then .supported |= false else . end' "$${NEW_CSV}"
+		yq -Yi '.spec.installModes[] |= if .type=="SingleNamespace" then .supported |= false else . end' "$${NEW_CSV}"
+		yq -Yi '.spec.installModes[] |= if .type=="MultiNamespace" then .supported |= false else . end' "$${NEW_CSV}"
+		yq -Yi '.spec.installModes[] |= if .type=="AllNamespaces" then .supported |= true else . end' "$${NEW_CSV}"
+		sed -ri 's|operatorframework.io/suggested-namespace: eclipse-che|operatorframework.io/suggested-namespace: openshift-operators|' "$${NEW_CSV}"
+
+		# Enable by default devWorkspace engine in `next-all-namespaces` channel
+		CSV_CR_SAMPLES=$$(yq -r ".metadata.annotations[\"alm-examples\"] | \
+				fromjson | \
+				( .[] | select(.kind == \"CheCluster\") | .spec.devWorkspace.enable) |= true" $${NEW_CSV} |  sed -r 's/"/\\"/g')
+		yq -riY ".metadata.annotations[\"alm-examples\"] = \"$${CSV_CR_SAMPLES}\"" $${NEW_CSV}
+
+		# Set next-all-namespaces channel name
+		ANNOTATION_METADATA_YAML="$${NEXT_BUNDLE_PATH}/metadata/annotations.yaml"
```

Comment:
I thought operator-sdk should set it to `next-all-namespace`

Reply:
no, by default we have next.

Reply:
But why? Where does it come from?

Reply:
https://github.com/eclipse-che/che-operator/blob/main/Makefile#L26
Comes from here. We can left as it and when all-namespaces will be by default just change there the default channel